libmscore/musescoreCore.h

```diff
@@ -34,7 +34,7 @@ class MuseScoreCore
       Score* currentScore() const        { return cs;     }
       void setCurrentScore(Score* score) { cs = score;    }
 
```

Comment:
what is the reason of bool* here?

I suggest to use simple `bool noToAll = false`

Reply:
The `savePng` and `saveSvg` functions ask the user about overwriting any existing files. If the user chose "Replace All" or "Skip All", that choice would formerly only be remembered inside that single call to the `saveSvg` or `savePng` function; if the function would be called the next time (for the next selected score) the user's choice will be forgotten, and the user will be asked again. 
To solve that, I use a pointer and pass it (via `saveAs`) to `savePng` and `saveSvg`, to make sure the `ExportDialog` and all calls to `savePng` or `saveSvg` are using the same bool, so if the user chooses "Replace All" or "Skip All", he won't be asked again. 
(Hope this explanation is clear ;-) )

Reply:
The `savePng` and `saveSvg` functions ask the user about overwriting any existing files. If the user chose "Replace All" or "Skip All", that choice would formerly only be remembered inside that single call to the `saveSvg` or `savePng` function; if the function would be called the next time (for the next selected score) the user's choice will be forgotten, and the user will be asked again. 
To solve that, I use a pointer and pass it (via `saveAs`) to `savePng` and `saveSvg`, to make sure the `ExportDialog` and all calls to `savePng` or `saveSvg` are using the same bool, so if the user chooses "Replace All" or "Skip All", he won't be asked again. 
(Hope this explanation is clear ;-) )

Reply:
ah, I see now, thx :)

It's quite confusing actually, perhaps would be bettter to create enum here and return it's values as a result

`virtual SavePolicy saveAs`

Reply:
Yes, that sounds like a good idea! I will try that soon (I'm quite busy these days!).
The saveAs function also returns a bool to indicate whether the export was successful. If that bool is never used, I can just change the function to return something like `SavePolicy`. But if the bool _is_ needed, shall I return a struct like this?
```
struct SaveAsResult {
    bool success;
    SavePolicy savePolicy
}
```

Reply:
yes, struct would make sense in this case

Reply:
I decided to create an `enum SaveReplacePolicy`, but still use pointers, because if I would pass the SaveReplacePolicy via the return value in a struct, quite many changes are necessary in other places, where only the `bool` about success is useful. I think that would cause many complications. 
Is that okay?

Reply:
Yeah, it's ok